### ISSUE TITLE
user_profile.bat Executed Twice On new CMD 

### ISSUE THREAD

**Issue by Necrolis** [cmderdev/cmder#2254]
### Purpose of the issue
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question

### Version Information
Cmder Version: 1.3.14.982
ConEmu Version: 191012 preview
Windows 10 Pro 10.0.18363

### Description of the issue
Whenever a new cmd console is launched, `user_profile.bat` appears to be executed twice.

I have enabled pageant as an ssh key provider by uncommenting the required lines in `user_profile.cmd`. Every time a new terminal is created I get the following output:
```
Starting ssh-pageant...
SSH_AUTH_SOCK='/tmp/.ssh-pageant-auth-sock'
Starting ssh-pageant...
SSH_AUTH_SOCK='/tmp/.ssh-pageant-auth-sock'

C:\cmder
λ
```
By enabled the debug logging in `init.cmd` I get the following output:
```
DEBUG(init.bat): Env Var - CMDER_ROOT=C:\cmder

DEBUG(init.bat): Env Var - debug_output=1

DEBUG(init.bat): Looking for Git install root...

DEBUG(:read_version): Env Var - git_executable=C:\cmder\vendor\git-for-windows\cmd\git.exe

DEBUG(:read_version): Env Var - GIT_VERSION_VENDORED=2.23.0.windows.1

DEBUG(:validate_version): ARGV[1]=VENDORED, ARGV[2]=2.23.0.windows.1

DEBUG(:parse_version): ARGV[1]=VENDORED, ARGV[2]=2.23.0.windows.1

DEBUG(:validate_version): Found Git Version for VENDORED: 2.23.0.windows.1

DEBUG(:read_version): Env Var - git_executable=C:\Program Files\Git\cmd\git.exe

DEBUG(:read_version): Env Var - GIT_VERSION_USER=2.24.0.windows.2

DEBUG(:validate_version): ARGV[1]=USER, ARGV[2]=2.24.0.windows.2

DEBUG(:parse_version): ARGV[1]=USER, ARGV[2]=2.24.0.windows.2

DEBUG(:validate_version): Found Git Version for USER: 2.24.0.windows.2

DEBUG(Comparing:):

DEBUG(USER:): 2.24.0.windows.2

DEBUG(VENDORED:): 2.23.0.windows.1

DEBUG(Using found Git '2.24.0.windows.2' from 'C:\Program Files\Git...):

DEBUG(Using Git from 'C:\Program Files\Git...):

DEBUG(init.bat): Preferring Windows commands

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\Program\ Files\\Git\\cmd - found=1

DEBUG(:enhance_path): Env Var END PATH C:\\Program\ Files\\Git\\cmd - found=1

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools

DEBUG(:enhance_path): Env Var C:\\Program\ Files\\Git\\cmd - found=1

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\Program\ Files\\Git\\mingw64\\bin - found=0

DEBUG(:enhance_path): Env Var END PATH C:\\Program\ Files\\Git\\mingw64\\bin - found=0

DEBUG(:enhance_path): Appending 'C:\Program Files\Git\mingw64\bin'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin

DEBUG(:enhance_path): Env Var C:\\Program\ Files\\Git\\mingw64\\bin - found=1

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\Program\ Files\\Git\\usr\\bin - found=0

DEBUG(:enhance_path): Env Var END PATH C:\\Program\ Files\\Git\\usr\\bin - found=0

DEBUG(:enhance_path): Appending 'C:\Program Files\Git\usr\bin'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin

DEBUG(:enhance_path): Env Var C:\\Program\ Files\\Git\\usr\\bin - found=1

DEBUG(init.bat): Env Var - git_locale="C:\Program & echo.
DEBUG(init.bat): Env Var - GIT_INSTALL_ROOT=C:\Program Files\Git

DEBUG(init.bat): Found Git in: 'C:\Program Files\Git'

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder\\vendor\\bin - found=0

DEBUG(:enhance_path): Env Var BEGIN PATH C:\\cmder\\vendor\\bin - found=0

DEBUG(:enhance_path): Prepending 'C:\cmder\vendor\bin'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\vendor\bin;C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin

DEBUG(:enhance_path): Env Var C:\\cmder\\vendor\\bin - found=1

DEBUG(:enhance_path_recursive): Env Var - add_path=C:\cmder\bin

DEBUG(:enhance_path_recursive): Env Var - position=

DEBUG(:enhance_path_recursive): Env Var - max_depth=1

DEBUG(:enhance_path_recursive): Adding parent directory - 'C:\cmder\bin'

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder\\bin - found=0

DEBUG(:enhance_path): Env Var BEGIN PATH C:\\cmder\\bin - found=0

DEBUG(:enhance_path): Prepending 'C:\cmder\bin'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\bin;C:\cmder\vendor\bin;C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin

DEBUG(:enhance_path): Env Var C:\\cmder\\bin - found=1

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder - found=0

DEBUG(:enhance_path): Env Var END PATH C:\\cmder - found=0

DEBUG(:enhance_path): Appending 'C:\cmder'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\bin;C:\cmder\vendor\bin;C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\cmder

DEBUG(:enhance_path): Env Var C:\\cmder - found=1

DEBUG(init.bat): Env Var - HOME=C:\Users\<redacted>

DEBUG(init.bat): Calling - C:\cmder\config\user_profile.cmd

Starting ssh-pageant...
SSH_AUTH_SOCK='/tmp/.ssh-pageant-auth-sock'
DEBUG(Using vendored Git '2.23.0.windows.1'...):

DEBUG(Using Git from 'C:\cmder\vendor\git-for-windows...):

DEBUG(init.bat): Preferring Windows commands

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder\\vendor\\git-for-windows\\cmd - found=0

DEBUG(:enhance_path): Env Var END PATH C:\\cmder\\vendor\\git-for-windows\\cmd - found=0

DEBUG(:enhance_path): Appending 'C:\cmder\vendor\git-for-windows\cmd'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\bin;C:\cmder\vendor\bin;C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\cmder;C:\cmder\vendor\git-for-windows\cmd

DEBUG(:enhance_path): Env Var C:\\cmder\\vendor\\git-for-windows\\cmd - found=1

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder\\vendor\\git-for-windows\\mingw64\\bin - found=0

DEBUG(:enhance_path): Env Var END PATH C:\\cmder\\vendor\\git-for-windows\\mingw64\\bin - found=0

DEBUG(:enhance_path): Appending 'C:\cmder\vendor\git-for-windows\mingw64\bin'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\bin;C:\cmder\vendor\bin;C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\cmder;C:\cmder\vendor\git-for-windows\cmd;C:\cmder\vendor\git-for-windows\mingw64\bin

DEBUG(:enhance_path): Env Var C:\\cmder\\vendor\\git-for-windows\\mingw64\\bin - found=1

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder\\vendor\\git-for-windows\\usr\\bin - found=0

DEBUG(:enhance_path): Env Var END PATH C:\\cmder\\vendor\\git-for-windows\\usr\\bin - found=0

DEBUG(:enhance_path): Appending 'C:\cmder\vendor\git-for-windows\usr\bin'

DEBUG(:enhance_path): END Env Var - PATH=C:\cmder\bin;C:\cmder\vendor\bin;C:\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\cmder\vendor\conemu-maximus5;C:\cmder\vendor\conemu-maximus5\ConEmu;C:\Python38\Scripts\;C:\Python38\;D:\VulkanSDK\1.1.130.0\Bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\TortoiseSVN\bin;C:\Program Files\TortoiseGit\bin;C:\Program Files\PuTTY\;C:\Program Files\Git\cmd;C:\Program Files\dotnet\;D:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\CMake\bin;C:\Users\<redacted>\AppData\Local\Microsoft\WindowsApps;C:\Users\<redacted>\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\<redacted>\AppData\Roaming\npm;C:\Users\<redacted>\.dotnet\tools;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\cmder;C:\cmder\vendor\git-for-windows\cmd;C:\cmder\vendor\git-for-windows\mingw64\bin;C:\cmder\vendor\git-for-windows\usr\bin

DEBUG(:enhance_path): Env Var C:\\cmder\\vendor\\git-for-windows\\usr\\bin - found=1

DEBUG(init.bat): Env Var - git_locale="C:\Program & echo.
DEBUG(init.bat): Env Var - GIT_INSTALL_ROOT=C:\cmder\vendor\git-for-windows

DEBUG(init.bat): Found Git in: 'C:\cmder\vendor\git-for-windows'

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder\\vendor\\bin - found=1

DEBUG(:enhance_path): Env Var BEGIN PATH C:\\cmder\\vendor\\bin - found=1

DEBUG(:enhance_path_recursive): Env Var - add_path=C:\cmder\bin

DEBUG(:enhance_path_recursive): Env Var - position=

DEBUG(:enhance_path_recursive): Env Var - max_depth=1

DEBUG(:enhance_path): Env Var INSIDE PATH C:\\cmder - found=1

DEBUG(:enhance_path): Env Var END PATH C:\\cmder - found=1

DEBUG(init.bat): Env Var - HOME=C:\Users\<redacted>

DEBUG(init.bat): Calling - C:\cmder\config\user_profile.cmd

Starting ssh-pageant...
SSH_AUTH_SOCK='/tmp/.ssh-pageant-auth-sock'

C:\cmder
λ
```

**Comment by mzellho** [comment]
Same for me, but in my case it is 4 times. You can also reproduce it by taking a plain installation and just inserting some echo into user_profile.cmd:

![image](https://user-images.githubusercontent.com/5822722/73919132-3ef09c00-48c3-11ea-8cbe-78e54be037a7.png)

**Comment by daxgames** [comment]
Sorry I can't reproduce this.

**Comment by daxgames** [comment]
![image](https://user-images.githubusercontent.com/7318053/74597166-3ff29d80-5028-11ea-9d0c-e08733e7a4d2.png)

**Comment by andyc56-mso** [comment]
I just did a plain installation of the cmder_mini.zip, placing this in user_profile.cmd:

echo Hello from user_profile.cmd

per mzellho.
Launching cmder from the right-click menu in File Explorer causes this prompt to be shown 3 times. This isn't exactly rocket science.

**Comment by wexman** [comment]
I have the same problem

**Comment by Necrolis** [comment]
@wexman Applying the fix in 251968b fixed my issues, you might want to apply it

**Comment by wexman** [comment]
Thanks, I will try that
